<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CloudViewer</class>
 <widget class="QWidget" name="CloudViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>267</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout"/>
  <action name="actionClearViewer">
   <property name="text">
    <string>ClearViewer</string>
   </property>
  </action>
  <action name="actionShowMainRefSyst">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ShowMainRefSyst</string>
   </property>
  </action>
  <action name="actionShowPCRefSyst">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ShowPCRefSyst</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>OpenKinect()</slot>
  <slot>CloseKinect()</slot>
  <slot>GrabKinect()</slot>
  <slot>TransformationChanged(TransformT)</slot>
 </slots>
</ui>
